contracts/ProfileGraph.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.0;
+
+import "./ProfileToken.sol";
+
+contract ProfileGraph is ProfileToken {
+
+    // data
+
+    //mapping(uint256 => uint256[]) public _followers;
+    //mapping(uint256 => uint256[]) public _following;
+
+    // events
+
+    event NewFollow(uint256 follower, uint256 following);
+    event UnFollow(uint256 follower, uint256 following);
```

Comment:
nit: I think we can just call them Follow / Unfollow to simplify

Reply:
Also maybe instead of follower we could call it target, and instead of following call it requester

Reply:
made it (follower,  followed) and (follower, unfollowed)